pocs/camera/camera.py

```diff
@@ -251,9 +353,105 @@ def get_thumbnail(self, seconds, file_path, thumbnail_size, keep_file=False, *ar
         image = fits.getdata(file_path)
         if not keep_file:
             os.unlink(file_path)
-        thumbnail = images.crop_data(image, box_width=thumbnail_size)
+        thumbnail = img_utils.crop_data(image, box_width=thumbnail_size)
         return thumbnail
 
+    def _fits_header(self, seconds, dark=None):
```

Comment:
Could probably move this to `fits_utils` file.

Reply:
I put it here because this is the function to generate the FITS header entries that (a) relate to the camera itself and (b) can be generated by the base class. There are similar functions in the derived classes that add additional camera related headers for which the implementation requires camera type specific code, and in both the base and Birger focuser classes.

I could move the base class methods to `fits_utils` but then I'd have to pass it the camera and focuser objects, and I'd still need FITS header generation methods in at least the derived camera and focuser classes to handle the hardware specific aspects.

Reply:
Makes sense, especially upon seeing what else was implemented further down in the PR.